app/routers/users.py

```diff
@@ -20,30 +20,35 @@
 # create user
 @router.post("/users/", response_model=UserResponse)
 async def create_user_endpoint(user: UserCreate, db: Session = Depends(get_db)) -> User:
-    return create_user(user=user, db=db)
+    result = await create_user(user=user, db=db)
+    return result
```

Comment:
아래와 같이 inline화 하셔도 충분할 것 같습니다. (변수 alloc에 비효율 방지 측면도 있을 것 같구요 ㅎㅎ)
다른 코드들도 동일하게 부탁드립니다!

```suggestion
    return await create_user(user=user, db=db)
```

Reply:
피드백 감사드립니다! 함수마다 전부 반영해서 커밋으로 올렸습니다!